src/main/java/org/acra/sender/EmailIntentSender.java

```diff
@@ -66,8 +66,12 @@ private String buildBody(@NonNull CrashReportData errorContent) {
 
         final StringBuilder builder = new StringBuilder();
         for (ReportField field : fields) {
+            String value = errorContent.get(field);
+            if (value != null) {

```

Comment:
Correct. There is a de-facto null-value, which is AcraConstants.NOT_AVAILABLE, but a collector returning null would be a bug.